### ISSUE TITLE
Filling out Description or Synonym as a non-admin user results in Error

### ISSUE THREAD

**Issue by shollorangehivede** [featdd/dpn_glossary#208]
Typo3-Error is: 
1: recordEditAccessInternals() check failed. [ERROR: The "languageField" field named "sys_language_uid" was not found in testing record!])

Possible reason: 
Missing 'sys_language_uid' in 'type' in Configuration/TCA/tx_dpnglossary_domain_model_description.php and Configuration/TCA/tx_dpnglossary_domain_model_synonym.php

Tested Typo3-Version: 11.5.33
Tested DNP Glossary Version: 5.1.0

**Comment by featdd** [comment]
Hi @shollorangehivede,

how do I reproduce this error?
The field is of course [defined here](https://github.com/featdd/dpn_glossary/blob/v5.1.0/Configuration/TCA/tx_dpnglossary_domain_model_synonym.php#L30), I've never encountered this error before.

Greetings
Daniel

**Comment by shollorangehivede** [comment]
Thank you for the quick answer - in our case, we updated the glossary (which was inactive for the time being) to the latest version and set it all up. Then we added user rights for an editor-user-group and tested the setup. All was working fine as an admin but when switching in the role of an editor, the error was shown when entering a description or synonym.
A colleague suggested to add the sys_language_uid to the 'types' in line 27. 
We got a work-around by overriding the TCA:

```<?php

use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;

//Add Language Field to DPN Glossary Description and Synonym Element

$GLOBALS['TCA']['tx_dpnglossary_domain_model_description']['palettes']['hiddenLanguagePalette'] = [
    'showitem' => 'sys_language_uid, l10n_parent',
    'isHiddenPalette' => true
];
$GLOBALS['TCA']['tx_dpnglossary_domain_model_synonym']['palettes']['hiddenLanguagePalette'] = [
    'showitem' => 'sys_language_uid, l10n_parent',
    'isHiddenPalette' => true
];

ExtensionManagementUtility::addToAllTCAtypes('tx_dpnglossary_domain_model_description', '--palette--;;hiddenLanguagePalette', '1');
ExtensionManagementUtility::addToAllTCAtypes('tx_dpnglossary_domain_model_synonym', '--palette--;;hiddenLanguagePalette', '1');
```

**Comment by featdd** [comment]
Hi @shollorangehivede,

I've fixed this here: 42e53f8c37be4040f4db111745edaefd9840ae3c

Greetings
Daniel
